CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 * Removes unnecessary white spaces in the bid result screen - yuki24
 * Updates the text style and order for the auction timer - yuki24
 * Registration screen sends mutation for creating a bidder/credit card - sweir27
+* There should be more space visually between conditions of sale checkbox and place bid button - yuki24
```

Comment:
I think this should go under `### master`

Reply:
Right, a new version just case out and this new entry was left in the wrong spot.